src/base.html

```diff
@@ -306,7 +306,7 @@
                   as the main headline.
                 -->
                 {%- if not "\x3ch1" in page.content -%}
-                  <h1>{{ page.title | default(config.site_name, true)}}</h1>
+                  <h1>{{ page.title | default(e(config.site_name), true)}}</h1>
```

Comment:
Doesn't work for me, I tried and think it should be `{{ page.title | default(config.site_name | e, true) }}`

Reply:
You are correct, I forgot to test that.